core/src/main/java/org/bitcoinj/quorums/ChainLocksHandler.java

```diff
@@ -92,6 +92,10 @@ public void close() {
         super.close();
     }
 
+    private boolean isInitialized() {
+        return blockChain != null;
+    }
```

Comment:
There is probably a race condition where some methods in this class are called before the blockChain object is initialized.